test/common/testlib.py

```diff
@@ -823,6 +833,10 @@ def setUp(self):
             if dhcp:
                 machine.dhcp_server()
 
+            # By default disable preloading on the default OS (the one with debug build)
```

Comment:
Please put the rationale here into the comment, and also point to test/image-prepare `build_and_install()` where the debug build is enabled.